src/types.ts

```diff
@@ -3806,6 +3806,27 @@ export interface LevelOpt {
 	[sym: string]: any,
 }
 
+export interface IsometricLevelOpt {
+	/**
+	 * Width of each block.
+	 */
+	tileWidth: number,
```

Comment:
Do you think this can be `width` and `height` to be consistent with `addLevel`? I do think `tileWidth` might be a better name, maybe we can deprecated `width` on both places and move to a more explicit name

Reply:
@slmjkdbtl yeah definitely! I did that just to make it easier for me to grasp about the code since it was my first time viewing the codebase. I will mirror property names from `addLevel` right now.